tests/test_config.py

```diff
@@ -223,6 +224,78 @@ def test_not_found_default_connection_from_evn_variable(test_root_path):
     )
 
 
+def test_correct_updates_of_connections_on_setting_default_connection(
+    test_snowcli_config, snowflake_home
+):
+    from snowflake.cli.api.config import CONFIG_MANAGER
+
+    connections_toml = snowflake_home / "connections.toml"
+    connections_toml.write_text(
+        """[asdf_a]
+    database = "asdf_a_database"
+    user = "asdf_a"
+    account = "asdf_a"
+    
+    [asdf_b]
+    database = "asdf_b_database"
+    user = "asdf_b"
+    account = "asdf_b"
+    """
+    )
+    config_init(test_snowcli_config)
+    set_config_value(section=None, key="default_connection_name", value="asdf_b")
+
+    def assert_correct_connections_loaded():
```

Comment:
Used 2 times:
* after setting `default_connection_name` to be sure that everything is ok in the memory at that moment
* after reinit of the config from the file to be sure that loading from file still is ok

Reply:
What do you think about reflecting this comments in test code by splitting in functions?
This may help in future read the code without searching for PR.

Reply:
IMO it won't help in the case of this test. IMO it only increase complexity of reading.

I added comments to PR only to simplify your reviews but IMO they are not required to read the code because these are not complicated changes.

Anyway I added one more comment line in the test case to make it more explanatory.